lib/states/put_money.rb

```diff
@@ -0,0 +1,53 @@
+module States
+  class PutMoney < Base
+    def action
+      puts I18n.t(:put_money_message)
+      return unless account_have_cards?(cards)
+
+      select_card_step
+    end
+
+    def next
+      MenuAccount.new(@context)
+    end
+
+    private
+
+    def read_amount_step(selected_card_index, current_card)
+      input_amount = read_input_with_title(I18n.t(:put_amount_message)).to_i
+      return unless amount_valid?(input_amount)
+
+      read_tax_step(selected_card_index, current_card, input_amount)
+    end
+
+    def select_card_step
+      print_cards(cards)
+      selected_card_index = read_input.to_i
+      return unless card_index_valid?(selected_card_index)
```

Comment:
I see three files where you are using this code. I remember I told you to fix it when I shared my screen and your code.